native-rating/implementation.md

```diff
@@ -100,28 +105,19 @@ affect the output and it is advised to keep a backup if you're not using source
 If you are using Typescript for your app, configuration will be stored in `./tsconfig.json` but again, it's advised that
 you don't modify this if you can avoid it. As before, it's worth keeping a backup if you're not using source control.
 
-## Implementation
-
-When the app shell is built and configured, you can [build your rating logic](./rating-logic.md). The linked document has instructions
-for building the logic from scratch, but the Connect CLI gives you a shell function to start with, located either at `./src/implementation/rate-shipments.js` or `./src/implementation/rate-shipments.ts`, depending on whether you chose Typescript or Javascript
-as your language.
-
-Any data necessary for testing can be stored in the `./src/demo-data.ts` file described in the configuration section of this document.
-Rates should go in the `rates` array, variables in the `variables` array, and zones (if you're using zones) in the `zones` array.
-
 ## Testing
 
 The Connect tooling provides a simple test server that you can use to test your app. To start it, run the following commands:
 
 ```bash
-npm run build   # Run before first start and after any data changes
+npm run build   # Run before first start and after any other Connect changes
```

Comment:
Should this be "other _code_ changes"? What are "Connect changes"?

Reply:
I changed this to `non-rating` because `code` would be too generic.  You _don't_ need to build after rating code changes but you _do_ need to build after any other code changes.